examples/example_linearity.ipynb

```diff
@@ -0,0 +1,217 @@
+    "Here outlines a basic linearity experiment where one sample (and therefore one vial location) is injected at varying injection volumes. \n",
+    "* Varies injection volume within a range in a defined incerement\n",
+    "* Samples in sample set defined in for loop, varying the injection volumne"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e369564f-3139-483e-bfc6-425124bd3706",
+   "metadata": {},
+   "source": [
+    "# Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2efb19df-2799-4946-89a1-291d1c6f7f36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from OptiHPLCHandler import EmpowerHandler\n",
+    "handler=EmpowerHandler(project=\"SLCB_2023_DEV\",address=\"http://epdkhqr01018.corp.novocorp.net:3076/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e67925bb-e6b6-4dd9-a96e-b9831b46bc6c",
+   "metadata": {},
+   "source": [
+    "# Configuration of methods, nodes, systems and plates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "872eee98-e52c-4ee1-a22e-ed3196837f73",
+   "metadata": {},
+   "source": [
+    "Using get attributes to: obtain method from list_method, node from list_nodes, system from list_systems within defined node and define the plate type from list_plates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "06e37043-9f51-4efc-87df-6254cdb12e12",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'handler' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[0;32m/tmp/ipykernel_1845/1937024686.py\u001b[0m in \u001b[0;36m<cell line: 1>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mlist_methods\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mhandler\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mGetMethodList\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0mmethod\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mlist_methods\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34mf\"Method Set Name: {method}\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0mlist_nodes\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mhandler\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mGetNodeNames\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mNameError\u001b[0m: name 'handler' is not defined"
```

Comment:
When everything is corrected, restart kernel and run all cells again, then save. Then we have the correct output